serverFiles/models/articles.models.js

```diff
@@ -18,14 +18,43 @@ function selectArticlesById(article_id) {
     });
 }
 
-function selectAllArticles() {
+function selectAllArticles(filterBy, sortBy, orderBy) {
+  const validSortBy = ['comment_count', 'author', 'title', 'article_id', 'topic', 'created_at', 'votes', 'article_img_url']
+    if(!validSortBy.includes(sortBy)) {
+      return Promise.reject({status:400,msg:'Bad request'})
+    }
+
+  const validFilterBy = ['mitch', 'cats', 'paper']
+    if (filterBy) {
+      if (!validFilterBy.includes(filterBy)) {
+        return Promise.reject({status:400,msg:'Bad request'})
+      }
+    }
+
+    const validOrderBy = ['asc', 'desc'];
+    if (orderBy) {
```

Comment:
As you're setting a default value for orderby and sortby in the model would the first if block ever trigger? if(orderBy)